hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1631,6 +1631,24 @@ possible configurations would overwhelm and obscure the important.
       Possible values are 'simple' (no authentication), and 'kerberos'.
     </description>
   </property>
+  <property>
+    <name>hbase.security.authentication.ui.metrics.protected</name>
+    <value>false</value>
```

Comment:
Isn't this changing the default behavior? I am assuming by default only admins are allowed to hit the endpoints.

Reply:
You're right. It is changing the default behavior.
I think metrics endpoints have more or less security vulnerabilities than `/conf` endpoint.
However, `/conf` endpoint can already be accessed by non admin users by default.
Therefore, I think this makes more sense.

But if HBase code guidelines insist the default behavior should not be changed, tell me, I also agree with that and make it not to change the default behavior.

Reply:
IMO we should not change default behavior for things related to security, as users will always have an option to set this explicitly if they need so.